app_dev.yaml

```diff
@@ -306,7 +306,6 @@ skip_files:
 - third_party/python_libs/oauthlib/
 - third_party/python_libs/requests_oauthlib/
 - third_party/python_libs/uritemplate/
-- third_party/python_libs/google/api_core/
```

Comment:
Removed this path as per request in [this comment](https://github.com/oppia/oppia/pull/11093#issuecomment-720003134).